lib/Conversion/TorchToTcp/TorchToTcp.cpp

```diff
@@ -49,6 +60,15 @@ class ConvertTorchToTcp : public ConvertTorchToTcpBase<ConvertTorchToTcp> {
 
   void runOnOperation() override {
     MLIRContext *context = &getContext();
+    RewritePatternSet patterns(context);
+
+    // The strings in the `convertTorchOps` ArrayRef don't exist during the call
```

Comment:
`The strings in the `convertTorchOps` ArrayRef don't exist` is weirdly phrased IMO; is it correct to say that usually the default constructor is called which means `convertTorchOps` is usually unset.

Reply:
Rephrased as suggested.